src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -2069,6 +2095,42 @@ public void NonRefStructContainer(string type)
                 Diagnostic(ErrorCode.ERR_RefFieldInNonRefStruct, "F").WithLocation(4, 13));
         }
 
+        [Theory]
+        [InlineData("class")]
+        [InlineData("struct")]
+        [InlineData("record")]
+        [InlineData("record struct")]
```

Comment:
Do we also have a test for a `ref struct` case?

Reply:
Looks like this one covers it:

https://github.com/dotnet/roslyn/blob/4a39725ac8b891eabd27585df9cebe4d97f9970f/src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs#L669